### PR TITLE
Added color picker to tinymce [stage-3]

### PR DESCRIPTION
**Pull request by duckbeartrap**
## Description
Added colorpicker plugin to tinymce.

## Motivation and Context
https://trello.com/c/czQqbQ7Y/65-add-tinymce-to-apps-2

## How Has This Been Tested?
N/A

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -85,6 +85,9 @@
   <script src="bower_components/angularjs-slider/dist/rzslider.min.js"></script>
   <script src="bower_components/tinymce/tinymce.min.js"></script>
   <script src="bower_components/angular-ui-tinymce/src/tinymce.js"></script>
+  <script src="bower_components/tinymce/plugins/colorpicker/plugin.js"></script>
+  <script src="bower_components/tinymce/plugins/wordcount/plugin.js"></script>
```
Comment (olegrise) [inline web/index.html:89]: @duckbeartrap what is this for?

Comment (duckbeartrap) [reply web/index.html:89]: These are the tinymce plugins for colorpicker. This is the way they imported them in widget-text.

Comment (duckbeartrap) [reply web/index.html:89]: Oh, I added the wordcount plugin too. Let me fix this.

**Comment by olegrise** [comment]
@duckbeartrap please don't forget to add `[stage-3]` to every commit message. The `[stage-0]` is used by our designers and we should be careful not to break their environment.

**Pull request merged**